internal/controlplane/handlers_projects.go

```diff
@@ -267,35 +265,6 @@ func (s *Server) DeleteProject(
 			"project does not allow project hierarchy operations")
 	}
 
-	// Gather the metadata needed for the project deletion
-	// This will then be used to create the audit log that helps with telemetry
-	var (
-		profilesCount int64
-		reposCount    int64
-		entitlements  []db.Entitlement
-	)
-
-	g, ctx := errgroup.WithContext(ctx)
-	g.Go(func() error {
-		var err error
-		profilesCount, err = qtx.CountProfilesByProjectID(ctx, projectID)
-		return err
-	})
-	g.Go(func() error {
-		var err error
-		reposCount, err = qtx.CountRepositoriesByProjectID(ctx, projectID)
-		return err
-	})
-	g.Go(func() error {
-		var err error
-		entitlements, err = qtx.GetEntitlementsByProjectID(ctx, projectID)
-		return err
-	})
```

Comment:
If these calls aren't used anymore, can you clean these up from the database queries?

Reply:
They are, in `internal/projects/deleter.go`